### PR TITLE
Fix locations for 50-50 molecule

### PR DESCRIPTION
**Pull request by KimberlyMunoz**
Moving a few files as part of a restructuring and adding fixes from #1063.

## Review
- @anselmbradford 
- @jimmynotjim 
- @sebworks

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -82,6 +82,7 @@ docs/*/pdfreactor-fonts.html
 *.egg-info
 .installed.cfg
 static/
+static_built/
```
Comment (anselmbradford) [inline .gitignore]: Should be

**Comment by anselmbradford** [comment]
Fix the gitignore path, otherwise :+1:

**Comment by anselmbradford** [comment]
:+1:

**Comment by jimmynotjim** [comment]
:+1:

**Pull request merged**